src/Workspaces/Core/MSBuild.BuildHost/Build/ProjectBuildManager.cs

```diff
@@ -123,7 +123,10 @@ public ProjectBuildManager(ImmutableDictionary<string, string> additionalGlobalP
             }
             else
             {
-                var projectCollection = new MSB.Evaluation.ProjectCollection(AllGlobalProperties);
+                var projectCollection = new MSB.Evaluation.ProjectCollection(
```

Comment:
Can we create a test for this?

Reply:
How were you thinking we would test this?  Are we able to test the contents of the binlog?

Reply:
Ah, think I figured this out.